src/current/cockroachcloud/export-logs-advanced.md

```diff
@@ -19,17 +19,19 @@ To configure and manage log export for your CockroachDB {{ site.data.products.ad
 https://cockroachlabs.cloud/api/v1/clusters/{your_cluster_id}/logexport
 ~~~
 
-Access to the `logexport` endpoint requires a valid CockroachDB {{ site.data.products.cloud }} [service account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) with the appropriate permissions (`admin` privilege or Cluster Admin role).
+Access to the `logexport` endpoint requires a valid CockroachDB {{ site.data.products.cloud }} [service account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) assigned one of the following [roles]({% link cockroachcloud/managing-access.md %}#edit-roles-on-a-service-account):
 
-The following methods are available for use with the `logexport` endpoint, and require the listed service account permissions:
+- [Organization Admin]({% link cockroachcloud/authorization.md %}#org-administrator)
```

Comment:
This is the wording in our docs, unless it changed?

```suggestion
- [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator)
```

Reply:
@Amruta-Ranade ,  is it ok I modify the [authorization page](https://deploy-preview-18456--cockroachdb-docs.netlify.app/docs/cockroachcloud/authorization#org-administrator) to use the correct role name -`Organization Admin`? That is what I am seeing in production: 
<img width="577" alt="Screenshot 2024-04-10 at 3 19 52 PM" src="https://github.com/cockroachdb/docs/assets/58752716/4d0437b0-b95d-41c9-98f3-a4f7a97b8132">

Reply:
Will defer to @mdlinville since it's security-related

Reply:
I discussed with @mdlinville that I will modify this PR to match the existing docs. See [this slack thread](https://cockroachlabs.slack.com/archives/C01CD39JX8E/p1712849732265149) for discussion about updating the role name in bulk in another PR.